pkg/model/build_settings.go

```diff
@@ -0,0 +1,11 @@
+package model
+
+var DefaultMaxBuildSlots = 1
+
+type UpdateSettings struct {
```

Comment:
re: Update vs Build naming

if it's called "update" does that mean people will think they can set max_parallel_updates=1 and that will make pods come up in serial? (i.e., that it will force Tilt to wait until pod A is ready before it even starts building image B?) i'm not sure if it matters but it's something to watch for

Reply:
oof. i... don't know. maybe? but on the flip-side, the word "build" is less and less accurate as we get into live_update, local_resource, etc. I think we've settled on "update" as the verb for that? but also 🤷‍♀️

Reply:
hmm yeah I'm not sure -- I'll make sure to clarify in the docs